src/actions/modal.actions.ts

```diff
@@ -0,0 +1,35 @@
+import { Dispatch } from 'redux';
+
+export namespace Modal {
+
+  export namespace Actions {
+
+    export class Open {
+      static type = 'MODAL_OPEN';
+      type = Open.type;
+      constructor(public payload: {
+        message: string,
+        open: boolean,
+        title: string,
+      }) {}
+    }
+
+    export class Close {
+      static type = 'MODAL_CLOSE';
+      type = Close.type;
+      constructor(public payload: {
+        button?: string,
```

Comment:
It doesn't look like this `button` field is used anywhere. I'm guessing it was added to potentially support confirm dialogs, but probably best to remove it for now to avoid confusion.